runtime/runtime/src/lib.rs

```diff
@@ -168,17 +132,17 @@ impl Runtime {
     }
 
     /// Subtracts the storage rent from the given account balance.
-    fn apply_rent(account_id: &AccountId, account: &mut Account, block_index: BlockIndex) {
+    fn apply_rent(&self, account_id: &AccountId, account: &mut Account, block_index: BlockIndex) {
         use primitives::serialize::Encode;
```

Comment:
Not sure why these imports are inside the function

Reply:
It is a way to de-clutter the code, by making the header of the file smaller. Because `primitives::serialized::Encode` is only used in this one method, there is no need to import it for the entire file.